https://issues.redhat.com/browse/ACM-26413 Console KI (#8506)
open-cluster-management/rhacm-docs: release_notes/known_issues_console.adoc

For more about deprecations and removals, see xref:../release_notes/acm_deprecate_remove.adoc#deprecations-removals-acm[Deprecations and removals for {acm-short}].

[#follow-external-link-wrong-page]

> == Following an external link opens the wrong page

Reviewer: ```suggestion
== External links to certain versions of the integrated console lead to incorrect pages
```

Reviewer: This is a bit confusing to me. Following an external link.... any external link, and external link from where???

Author: Should be any external link that leads to the console. Could be from anywhere.

Can you confirm @KevinFCormier ?

Reviewer: Now that I read the example, I am not sure where the "external" comes from.

But it seems we need to make it clearer 1. The OCP Version and 2. The the tabs in the console are what open to incorrect pages.

I find this quite confusing as of now.

Author: The internal links should work I think. Meaning the ones in the console navigation.

I'll add the OCP versions that are affected.

The tabs in the console are just one of several ways to reach incorrect pages from what I understand.

Reviewer: Can we shorten this:

[#external-link-wrong-page]

> If you follow an external link that leads to the console, open a link in a new tab, enter a URL manually, or refresh a page, part of the URL might be lost. As a result, you see the wrong console page. For example, a link to the *Governance Policies* tab might open the *Governance Overview* tab instead.

Reviewer: Do you mean "open a new tab" then enter the URL?

Open an link in a new tab... what link???

Author: Yes, opening a new tab and entering the URL.

The link is any link that leads to the console, not a specific link.

Reviewer: "An external link to the integrated console leads to incorrect links within the console"?

An external link that leads to the console -- this seems broad. 

And I think we need to say early that it is only certain versions.

Author: I added the specific OCP versions that are affected

> If you follow an external link that leads to the console, open a link in a new tab, enter a URL manually, or refresh a page, part of the URL might be lost. As a result, you see the wrong console page. For example, a link to the *Governance Policies* tab might open the *Governance Overview* tab instead.

Reviewer: When reading that first line, I felt it was a series of steps. 

When you do this, then this, then this. Some changes, maybe?

_____

If you follow an external link that leads to **certain versions** of the console, open an external link in a new tab, enter a URL **to the console manually**, or refresh a page t**o the console,** part of the URL might be lost.

____

It doesn't have to be this but seeing if we can make it clearer. This call also just be bullets.

Author: I changed it to bullets and did something similar to your suggestion. I think it would be too long without the bullets. Let me know what you think.

> To resolve the issue, upgrade to {ocp} version 4.20.8 or later.

Reviewer 2: It's fixed in earlier releases than this... I would list all 3 versions.

Author: This is for 2.15 doc, we need separate PRs for the other versions since they are in other branches. I'll list the other OCP versions in each of those.

Reviewer 2: ACM supports OCP n-2, and customers are often not on the latest OCP release. So ACM 2.15 is supported with OCP 4.18. I don't want to give the impression that they need OCP 4.20 to get this fix.

Author: I see, thanks for explaining. I'll add the other versions!

> Alternatively, you might be able to access the correct page if you reload the page. You can also navigate to the correct page from the console by following the appropriate links in the navigation menu, or by using the search function.

Reviewer: ```suggestion
Alternatively, you can reload to try to access the correct page. You can also navigate to the correct page from the links in the console navigation menu, or use the search function.
```

Reviewer: I think Search might be capitalized?

Author: I've seen both in the doc. Not sure tbh. I applied the other suggestions.

I think if we refer to the search function as a generic function, I'd probably leave it lower case. But if we refer to the component/feature, perhaps we should capitalize?
